lib/parser.js

```diff
@@ -45,7 +45,7 @@ var DEPS_EASY_GAV_STRING_REGEX = RegExp('(["\']?)([\\w.-]+):([\\w.-]+):([\\w\\[\
 var DEPS_HARD_GAV_STRING_REGEX = RegExp(DEPS_KEYWORD_STRING_PATTERN + '(?:\\((.*)\\)|(.*))');
 var DEPS_ITEM_BLOCK_REGEX = RegExp(DEPS_KEYWORD_STRING_PATTERN + '\\(((["\']?)(.*)\\3)\\)[ \\t]*\\{');
 var DEPS_EXCLUDE_LINE_REGEX = RegExp('exclude[ \\t]+([^\\n]+)', 'g');
-var PLUGINS_LINE_PATTERN = RegExp('(id|version)(?:[ \\t])(["\']?)([A-Za-z0-9.]+)\\2', 'g');
+var PLUGINS_LINE_PATTERN = RegExp('(id|version)[ \\t]*\\(?(["\']?)([A-Za-z0-9.]+)\\2\\)?', 'g');
```

Comment:
I was considering to add two expressions for this, basically one with and one without the parentheses, but I think this will be good enough anyways. :D